indico/modules/categories/client/js/calendar.js

```diff
@@ -9,18 +9,69 @@
 
 import {Calendar} from '@fullcalendar/core';
 import dayGridPlugin from '@fullcalendar/daygrid';
+import moment from 'moment';
+import React from 'react';
+import ReactDOM from 'react-dom';
 
+import CalendarSingleDatePicker from 'indico/react/components/dates/CalendarSingleDatePicker';
```

Comment:
```suggestion
import {CalendarSingleDatePicker} from 'indico/react/components';
```

we re-export all those components from *components*

Reply:
Done 👍